### ISSUE TITLE
Allow webp file uploads

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#2426]
WordPress core allows them as of 5.8, so should we: https://make.wordpress.org/core/2021/06/07/wordpress-5-8-adds-webp-support/

**Comment by JohnMcLeod13** [comment]
@SteelWagstaff Request came from end user on New Haven network, they had used webp files on their Reclaim hosting instance previously. https://pressbooks.zendesk.com/agent/tickets/18048
Is it as simple as adding the file extension to the upload settings on the network?

**Comment by SteelWagstaff** [comment]
Yes -- that will allow it. I think we could also add it as a permitted file type so that super admins don't have to take this step. Just opened a PR to do that.

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 4y 5m ago with no updates in the last 1y 2m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542
